bonusopdracht.js

```diff
@@ -0,0 +1,85 @@
+//7.4
+/*const popularBookTitle = document.createElement('h3');
+popularBookTitle.textContent = "The Great Gatsby";
+popularBookTitle.setAttribute('class', 'pink-background');
+const popularBookAuthor = document.createElement('p');
+popularBookAuthor.textContent = 'Door F. Scott Fitzgerald';
+
+//popularBookTitle.removeAttribute('class');
+const popularBook = document.getElementById('popular-book');
+popularBook.appendChild(popularBookTitle);
+popularBook.appendChild(popularBookAuthor);*/
+
+
+//1. Maak drie knoppen op de pagina: Sorteer op prijs, AmbiLight TV's en Uitverkochte exemplaren. Gebruik de code die je in opdracht 1b, 1c en 1d hebt gemaakt en schrijf dit om naar functies zodat je ze kunt aanroepen op het moment dat de buttons geklikt worden. Zorg ervoor dat de functies de uitkomsten in de de console loggen als de gebruiker op de bijbehorende knop klikt. Tip: lees hiervoor paragraaf 7.4 op EdHub eens door!
+// script voor 1b
+let btn = document.createElement("button");
+btn.innerHTML = "Uitverkochte Exemplaren";
+document.body.appendChild(btn);
+btn.onclick = function () {
+    alert("Uitverkochte Button is clicked");
+
+const uitVerkochtObjecten = inventory.filter((inventory) => {
+    return inventory.originalStock === inventory.sold
+
+})
+console.log(uitVerkochtObjecten);
+};
+// script voor 1c
+let btn1 = document.createElement("button");
+btn1.innerHTML = " Ambilight TV's ";
+document.body.appendChild(btn1);
+
+btn1.onclick = function () {
+    alert("Ambilight Button is clicked");
+
+    const ambiLightBeschikken = inventory.filter((inventory) => {
+
+        return inventory.options.ambiLight === true
+    })
```

Comment:
Deze had je nog iets kunnnen versimpelen door `=== true` weg te laten
```suggestion
    const ambiLightBeschikken = inventory.filter(inventory => inventory.options.ambiLight);
```